docs/gluon/index.md

```diff
@@ -82,7 +82,6 @@ def forward(self, F, inputs, tree):
                         for child in tree.children]
     #Recursively builds the neural network based on each input sentence’s
     #syntactic structure during the model definition and training process
-    …
```

Comment:
Was this breaking something or is your editor linting things?

Reply:
This was causing an warning, and when removed, I was able to remove that warning, but now I'm unable to reproduce so I've added it back in. However, this does look like it shouldn't be there. It's a UTF-8 ellipsis symbol.